apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetService.java

```diff
@@ -266,11 +269,13 @@ public void delete(@NonNull UUID id) {
     }
 
     @Override
-    public DatasetPage find(int page, int size, @NonNull DatasetCriteria criteria) {
+    public DatasetPage find(int page, int size, @NonNull DatasetCriteria criteria, List<SortingField> sortingFieldsList) {
```

Comment:
Valid date the sortingFieldsList param and maybe rename it to `sortingFields`

Reply:
Validation happens inside `sortingFactory.newSorting(sorting)`.
Renamed

Reply:
Ok, but this is a public method; other places can call it.